### PR TITLE
Updated Japanese localization.

### PR DESCRIPTION
**Pull request by zombine04**

### CODE REVIEW

File: QuickLookCard/scripts/mods/QuickLookCard/QuickLookCard_localization.lua [hunk 1]
```diff
@@ -249,110 +249,137 @@ return {
...
 	},
 	trait_gadget_cooldown_reduction = {
 		en = "ABRG", -- Combat Ability Regeneration
 		["zh-cn"] = "技回", -- 作战技能恢复
+		ja = "能回", -- 戦闘アビリティ回復
 	},
 	trait_weapon_trait_increase_impact = {
 		en = "IMPC", -- Impact (Melee)
 		["zh-cn"] = "冲击", -- 冲击（近战）
+		ja = "衝撃" -- 衝撃
```
Comment (deluxghost) [inline QuickLookCard/scripts/mods/QuickLookCard/QuickLookCard_localization.lua]: Please add the trailing comma here, so next guy wont need to remember that

**Comment by deluxghost** [comment]
I guess this mod is also the most weird and interesting translating experience for you

File: QuickLookCard/scripts/mods/QuickLookCard/QuickLookCard_localization.lua (hunk 1 above)
Comment (zombine04) [reply QuickLookCard/scripts/mods/QuickLookCard/QuickLookCard_localization.lua]: Added.

**Comment by zombine04** [comment]
> I guess this mod is also the most weird and interesting translating experience for you

I often wonder how to abbreviate some words into 2 letters XD

**Pull request merged**